pywbem_mock/_wbemconnection_mock.py

```diff
@@ -91,29 +91,49 @@ def _uprint(dest, text):
     """
     Write text to dest, adding a newline character.
 
-    Text must be a unicode string and must not be None.
-
-    If dest is a file path, the text is encoded to a UTF-8 Byte sequence and
-    is appended to the file (opening and closing the file).
+    Text may be a unicode string, or a byte string in UTF-8 encoding.
+    It must not be None.
 
     If dest is None, the text is encoded to a codepage suitable for the current
     stdout and is written to stdout.
+
+    Otherwise, dest must be a file path, and the text is encoded to a UTF-8
+    Byte sequence and is appended to the file (opening and closing the file).
     """
-    assert isinstance(text, six.text_type)
+    if isinstance(text, six.text_type):
+        text = text + u'\n'
+    elif isinstance(text, six.binary_type):
+        text = text + b'\n'
+    else:
+        raise TypeError(
+            "text must be a unicode or byte string, but is {0}".
```

Comment:
`text` is meant to be the name of the argument. Would we still capitalize it?

i can add showing the text.

Reply:
No. That was my mistake.   Looks strange but that is correct.  Showing the text is a judgement call.  Since we assume that it is probably NEVER going to happen. My tendencey is still to show as much as possible but I am not married to that perception.